content/blog/contributor-highlight-daniel-chambre/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Contributor Spotlight: Daniel Chambre"
+date: 2025-04-25T14:51:43+02:00
+draft: false
+image:
+    url: daniel.jpg
+    attribution:
+author: fdrees
+tags:
+ - kubernetes
+ - service-mesh
+ - dba
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Daniel Chambre (aka smiyc), a DBA at WienIT."
+---
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+
+Today we're meeting Daniel Chambre, better known under his handle: 
+[smiyc](https://github.com/smiyc/). Daniel is a DBA at WienIT, working part 
+remotely from "the outskirts of Vienna". He first learned about CNP by/for EDB 
```

Comment:
"by/for" here, not sold on this.
And, probably worth explaining that CNP was the closed-source Kubernetes operator for Postgres, from which CloudNativePG was split.
How about something like:
"He first learned about CNP, EDB's Kubernetes operator for Postgres, in late 2021. After CloudNativePG was spun off in XYZ ..."